content/articles/drug-labels-extraction-using-deep-learning/index.md

```diff
@@ -0,0 +1,224 @@
+author: wilkister-mumbi
+date: 2021-09-13T00:00:00-18:30
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/drug-labels-extraction-using-deep-learning/hero.png
+    alt: Drug Label Image
+---
+Optical Character Recognition (OCR) is a technology that can be used to extract readable text into a machine-encoded text. A large number of companies that process paper-based forms use OCR to extract texts from documents.
+<!--more-->
+Applying cutting-edge technologies to modern problems has enabled various problem-solving in healthcare thereby improving people's lives. Machine learning has been used in hospitals to identify patients faster, in disease identification and diagnosis, robotic surgery, and medical imaging diagnosis.
+
+One notable area of concern in healthcare that hasn't been explored a lot is in drug labels extraction. Optical Character Recognition (OCR) is a technology that can be used to extract readable/handwritten text into a machine-encoded text. 
+
+The OCR technology has been used on Google's visual translation service. Also, a large number of companies that process paper-based forms use OCR to extract texts from documents. Recently, most companies offering PDF services such as Adobe provide this feature on their app.  
+
+This technology is what our tutorial will be based on. We'll use OCR to enable us automatically extract drug labels/text from prescription medicine. 
+
+We'll specifically use the PaddleOCR, previously referenced in a paper known as [PP-OCR: A Practical Ultra Lightweight OCR System](https://arxiv.org/pdf/2009.09941v3.pdf). This is a model that was originally built by the engineers at [Baidu](https://www.baidu.com/). 
+
+### Prerequisites
+To understand this tutorial, you need:
+- To have either Google Colab or Jupyter Notebook installed.
+- You need to be familiar with machine learning.
+- You need to have Python and PaddleOCR installed.
+
+### Table of contents
+- [Installing PaddleOCR for Python](#installing-paddleocr-for-python)
+- [Applying PaddleOCR model to images](#applying-paddleocr-model-to-images)
+- [Extracting prescription medication labels using PaddleOCR](#extracting-prescription-medication-labels-using-paddleocr)
+- [Wrapping up](#wrapping-up)
+
+To go through with our drug label extraction, we need to perform three key things:
+
+- Install and import our dependencies.
+- Instantiate the model and detect.
+- Visualize the results.
+
+### Installing PaddleOCR for Python
+First, what we need to do is to open either your colab or notebook and install the dependencies. The dependency we want to install is the PaddleOCR. Please head on to Paddle OCR's Github [page](https://github.com/PaddlePaddle/PaddleOCR). 
+
+On the `Tutorial` section down below on the page, go to the `installation` link and click on it. On this page, you will be provided with a quick installation guide for the PaddleOCR.
```

Comment:
Under the `Tutorial` section, navigate to the `installation` sub-section. A thorough guide to install PaddleOCR is available here (possibly attach the link instead of asking the reader to go to another page and find the link)